integration_test/gce/gce_testing.go

```diff
@@ -1783,6 +1788,8 @@ type VMOptions struct {
 	// Optional. If missing, the default is e2-standard-4.
 	// Overridden by INSTANCE_SIZE if that environment variable is set.
 	MachineType string
+	// Optional. Used to specify any required accelerators
+	Accelerator string
```

Comment:
This is needed because of the extra flag for GPU VMs (`--maintenance-policy=TERMINATE`). Should we set this for all VMs since the testing VMs will be short-living anyway? Then we can combine this into `ExtraCreateArguments`.

Reply:
Why can't that go in `ExtraCreateArguments` too? e.g.

```go
options.ExtraCreateArguments = append(
    options.ExtraCreateArguments,
    fmt.Sprintf("--accelerator=count=1,type=%s", tc.gpu.model),
    "--maintenance-policy=TERMINATE")
```

Reply:
Done. Thanks!